it/CallHelpdeskControllerISpec.scala

```diff
@@ -90,8 +91,21 @@ class CallHelpdeskControllerISpec extends HelperSpec {
             submitCallOptionResponse.header(LOCATION).get shouldBe "/helpline/select-national-insurance-service"
           }
         }
+      }
+
+      "the national insurance option is selected on the which-service-are-you-trying-to-access page and the findMyNinoEnabled flag is set to true" in new FindMyNinoEnabled {
```

Comment:
can we have a regression test to show that the page goes to the appropriate place with the FS disabled for national-insurance please :)